backend-diagram-converter/cli/src/main/java/org/camunda/community/migration/converter/cli/AbstractConvertCommand.java

```diff
@@ -96,10 +96,10 @@ private void writeResults(
       }
     }
     if (csv) {
-      try (FileWriter fw =
-          new FileWriter(
-              determineFileName(new File(targetDirectory(), "conversion-results.csv")))) {
+      File csvFile = determineFileName(new File(targetDirectory(), "conversion-results.csv"));
+      try (FileWriter fw = new FileWriter(csvFile)) {
         converter.writeCsvFile(results, fw);
+        LOG_CLI.info("Created {}", csvFile.getName());
```

Comment:
Instead of `csvFile.getName()`, there should be only `csvFile` like in like 88.

Reply:
Ahh, that's where the `.\` comes from:

```
Created .\converted-c8-invoice.v2.bpmn 
Created .\converted-c8-reviewInvoice.bpmn 
Created .\conversion-results (1).csv 
```